### PR TITLE
Profiles: ens profile go to NFTs from avatar / cover

### PR DESCRIPTION
**Pull request by estebanmino**
Fixes RNBW-3410
Fixes RNBW-3409

## Final checklist

- [ ] Assigned individual reviewers?
- [ ] Added labels?
- [ ] Added e2e tests? if not please specify why
- [ ] If you added new files, did you update the CODEOWNERS file?

### CODE REVIEW

**Comment by estebanmino** [comment]
added images fade in as well
https://user-images.githubusercontent.com/12115171/165815942-93dfbfdf-bb66-42c2-a9c2-4fb82b60c86a.mp4

**Comment by estebanmino** [comment]
updated and now images aren't jumping around as the last video
https://user-images.githubusercontent.com/12115171/165833468-59225885-bd5f-4602-be3a-8783a1a745c0.mp4

**Review by jxom — APPROVED** [review]
lgtm, not sure why the polish merge is included in these changes too? do we need to merge in?

File: src/components/ens-profile/ProfileSheetHeader.tsx [hunk 1]
```diff
@@ -37,10 +45,87 @@ export default function ProfileSheetHeader({
...
+  );
+
+  const {
+    enableZoomOnPressAvatar,
+    enableZoomOnPressCover,
+    onPressAvatar,
+    onPressCover,
+    avatarUrl,
+    coverUrl,
+  } = useMemo(() => {
```
Comment (jxom) [inline src/components/ens-profile/ProfileSheetHeader.tsx:86]: does this need to be a big `useMemo`? can't we split this up?

**Comment by estebanmino** [comment]
oh I'll cherry pick

**Review by jxom — APPROVED** [review]

**Pull request merged**